content/blog/2020-cfp-extension.md

```diff
@@ -0,0 +1,10 @@
+Title: Bonus Round: CFP extended on overwhelming requests
+Date: 2020-08-15 15:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-cfp-extension
+Summary: We are extending our talks CFP till August 19
+
+We are getting a large number of talk submissions and requests from folks who were not able to complete their proposals in time. So, due to popular demand, we are extending our CFP deadline for talks until **August 19, 23:59 UTC+5:30**. If you were unable to submit your proposal on time, you now have **5 more days** to do it. 

```

Comment:
```suggestion
PyCon India 2020 call for proposals (CFP) received a tremendous amount of talks and workshops from the global community as a whole. The deadline for the CFP was 14 August 2020.

After the official closure of CFP, we are getting a large number of talk submissions and requests from folks who were not able to complete their proposals in time. So, due to popular demand, we are extending our CFP deadline for talks until **August 19, 23:59 UTC+5:30**. If you were unable to submit your proposal on time, you now have **5 more days** to do it now. 

Please take this time to submit new proposals, improve the ones you submitted based on the review received, and make sure to fulfill the submission criteria by all means. This will be our final extension to the call for proposals. Hope you make the most of it. 
```